refactor: Remove unnecessary debouncing in editor components

codeAntu opened the pull request:
<!--- Provide a formatted commit message describing this PR in the Title above -->
<!--- See our DEVELOPERS guide below: -->
<!--- https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format -->
# Closes #315 
<!--- Provide an overall summary of the pull request -->

### Changes
<!--- More detailed and granular description of changes -->
<!--- These should likely be gathered from commit message summaries -->
- <ONE>Removed unnecessary debouncing in editor components


### Related Issues
- Issue #315 
- Pull Request #<NUMBER>

### Author Checklist
- [ ] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `main` from `fork:branchname`

DianaLease commented on src/editors/editorsContainer/TemplateMarkdown.tsx:
```diff
@@ -13,18 +11,11 @@ function TemplateMarkdown() {
     if (value !== undefined) {
       setValue(value);
-      debouncedSetTemplateMarkdown(value);
+      setTemplateMarkdown(value); // Call setTemplateMarkdown directly
```
can you remove the comment please? same goes for each file, thanks!

codeAntu replied:
Yes , sure .

DianaLease reviewed (changes requested):
Can you update with main and resolve conflicts? Thanks!

codeAntu commented:
Hi @DianaLease,

I have removed the comments, updated the branch with the main branch, and resolved the conflicts. Please review the changes.

Thank you!
